fetch_data/sparql_data/create_connectivity_data_observation.py

```diff
@@ -32,9 +33,7 @@ async def create_connectivity_observation(wikibase_id: int) -> bool:
             require_sparql_endpoint=True,
         )
 
-        observation = await compile_connectivity_observation(
-            wikibase.sparql_endpoint_url.url
-        )
+        observation = await compile_connectivity_observation(wikibase)
```

Comment:
I did not expect this change based on the PR title.

Reply:
I needed to pass the full Wikibase into this method, so that I could specify the Wikibase in the logs